src/wallet/wallet.cpp

```diff
@@ -1842,12 +1846,17 @@ int CWallet::ScanForWalletTransactions(CBlockIndex* pindexStart, bool fUpdate)
                     ret++;
             }
 
-            ZCIncrementalMerkleTree tree;
+            ZCIncrementalMerkleTree sproutTree;
+            ZCSaplingIncrementalMerkleTree saplingTree;
             // This should never fail: we should always be able to get the tree
             // state on the path to the tip of our chain
-            assert(pcoinsTip->GetSproutAnchorAt(pindex->hashSproutAnchor, tree));
+            assert(pcoinsTip->GetSproutAnchorAt(pindex->hashSproutAnchor, sproutTree));
+            // TODO: (when hashFinalSaplingAnchor is added)
```

Comment:
It's already there - it's in the block header, which `pindex` contains.

Reply:
Ok I see it now. It was called hashFinalSaplingRoot not hashFinalSaplingAnchor.